GlotPress/gp-translation-helpers
# Add the `translation-row-editor-meta` template for the right sidebar

## Problem



In [this previous PR (Add some components into tabs in the right sidebar)](https://github.com/GlotPress/gp-translation-helpers/pull/148), I add tabs to the right sidebar. This PR depended on this https://github.com/GlotPress/GlotPress/pull/1531, because it adds a required filter.

## Solution



This new PR doesn't need the GlotPress filter, because it overwrites the `translation-row-editor-meta` template, using the `gp_tmpl_load_locations` filter.

## Testing instructions



You have to work with the content on the 4 tabs:

- Meta. You can work as you usually work on the current tab to test the functionality.
- Discuss. You can create new comments and/or reply to the existing ones.
- History. You can check the translation history on this tab.
- Other locales. You can check the translation of other locales made to this translation and copy the content to the translation textarea. The copied text doesn't overwrite the current translation, except you select the current translation before you copy the text.
